nshlib/nsh_session.c

```diff
@@ -169,6 +170,21 @@ int nsh_session(FAR struct console_stdio_s *pstate,
         }
     }
 
+#if CONFIG_NSH_DISABLE_ECHOBACK
```

Comment:
should we move the change before line 104?

Reply:
I don’t get it...
Could you explain that please?